src/main/metrics/push-gateway.ts

```diff
@@ -0,0 +1,41 @@
+import { Request } from '@automationcloud/request';
+import { inject, injectable, multiInject } from 'inversify';
+
+import { Config, config } from '../config';
+import { Logger } from '../logger';
+import { MetricsRegistry } from './registry';
+
+const DEFAULT_PUSH_GATE_URL = 'http://push-gateway.monitoring.svc.cluster.local:9091/metrics';
+@injectable()
+export class MetricsPushgateway {
```

Comment:
Since we separate the words in `PUSH_GATEWAY`, it'd make sense to make this one `MetricsPushGateway` as well.

Reply:
Oh yup, in fact when I double checked the official name of prometheus, it was `pushgateway` rather than `push-gateway`, so I've changed the class name but not the other parts. should we just align all with `push gateway` rather than `pushgateway`

Reply:
Yep, I'd go with `PushGateway`